Fabric.Identity.API/scripts/BenTestLog.txt

```diff
@@ -0,0 +1,4772 @@
+Error response from daemon: No such container: identity.integration.openldap
```

Comment:
Does this need to be checked in?

Reply:
Yes, this should be removed. I will take care of that.